src/common/routing/RouteDefs.jsx

```diff
@@ -1,7 +1,13 @@
 import App from "App";
+<<<<<<< HEAD
```

Comment:
You should never commit this file as you have not changes in this file.

Also committing and pushing with  a merge conflict is a CRIME, haha 😂